src/test/test_dash.cpp

```diff
@@ -38,6 +39,7 @@ extern void noui_connect();
 
 BasicTestingSetup::BasicTestingSetup(const std::string& chainName)
 {
+        SHA256AutoDetect();
```

Comment:
missing similar changes in bench_dash.cpp

Reply:
done